packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value = 0,
+  max = 100,
+  size = 'regular',
+  hideLabel,
+  children,
+  className,
+}: ProgressProps): JSX.Element => {
+  const normalizedValue = Math.round((value / max) * 100);
+
+  return (
+    <div
+      className={classy(c('progress'), m(size), className)}
+      role="progressbar"
+      aria-valuenow={value}
+      aria-valuemin={0}
```

Comment:
I wonder, do we have a usecase to also set a `min`?

For example, what if your progress is a part of 50 to a 100, so 50 would be 0%?

Reply:
Yes, I think that's exactly what happens, some implementations like [this](https://github.com/uber/baseweb/blob/master/src/progress-bar/progressbar.js#L99) sets it default to 0, but I don't see a problem in having the option.

I can default it to 0, what do you think?

Reply:
I think yes to be more future-proof, better to have this as optional prop defaulting to a `0`.

Reply:
I'd say implement it when we need it as it's not a breaking change, but I don't see why not do it now as it seems to be simple.

Reply:
I have added the min prop.